jsonnet/control-plane.libsonnet

```
@@ -9,20 +9,8 @@ function(params)
       _config+:: cfg.mixin._config,
     },
 
-    etcdPrometheusRule: {
-      apiVersion: 'monitoring.coreos.com/v1',
-      kind: 'PrometheusRule',
-      metadata: {
-        labels: cfg.commonLabels + cfg.mixin.ruleLabels,
-        name: 'etcd-prometheus-rules',
-        namespace: cfg.namespace,
-      },
-      spec: {
-        local r = if std.objectHasAll($.etcdMixin, 'prometheusRules') then $.etcdMixin.prometheusRules.groups else [],
-        local a = if std.objectHasAll($.etcdMixin, 'prometheusAlerts') then $.etcdMixin.prometheusAlerts.groups else [],
-        groups: a + r,
-      },
-    },
+    // etcd alerting and recording rules are handled by the etcd operator.
+    etcdPrometheusRule: null,
```

```suggestion
    etcdPrometheusRule:: null,
```

This will make the object hidden and thus easier to work within loops.